src/Features/Core/Portable/EditAndContinue/BidirectionalMap.cs

```diff
@@ -31,11 +31,11 @@ public BidirectionalMap(IEnumerable<KeyValuePair<T, T>> entries)
             Reverse = reverseMap;
         }
 
-        public static BidirectionalMap<T> FromMatch(Match<T> match)
+        public static BidirectionalMap<SyntaxNode> FromMatch(Match<SyntaxNode> match)
```

Comment:
Revert, not syntax node specific. Besides the containing class is ```<T>``` and this is a factory for it.